.gitignore

```diff
@@ -21,3 +21,8 @@ venv/
 # directories for dev and prod. Resource directories for prod are generated
 # via build.py and should not be committed to the repo.
 build/*
+
+# The pre_commit_linter.py script uses _check_docs_utils.py placed
+# in python-custom-checks directory under third_party directory.
+# Therefore it is not ignored.
+!third_party/python-custom-checks
```

Comment:
Sorry, no, this is not what I meant. I actually meant to locate the source file in third_party so that we do not have to ship it on github. Then modify the install scripts to download whatever is needed (pylint etc., or that specific file) and refer to it there.

That way, we don't ship/maintain code that is not ours (which is kind of the point of third-party in the first place).

Happy to chat about this if anything's not clear.

Reply:
Done as discussed. Thanks!